### PR TITLE
Fixed the command continuation characters that were missing and skipp…

### PR DESCRIPTION
**Pull request by k-dahl**
…ing the build process

Switched ipython to be v3.2 as 4.0 no longer appears to support profiles correctly

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ ENV SPARK_HOME /usr/local/src/spark-$SPARK_VERSION
 RUN \
     mkdir -p $SPARK_HOME &&\
     curl -s http://d3kbcqa49mib13.cloudfront.net/spark-$SPARK_VERSION.tgz | tar -xz -C $SPARK_HOME --strip-components=1 &&\
-    cd $SPARK_HOME \
+    cd $SPARK_HOME &&\
```
Comment (prabeesh) [inline Dockerfile:21]: LGTM

File: Dockerfile [hunk 2]
```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y build-essential \
     python-zmq
 
 RUN pip install py4j \
-    ipython[notebook] \
+    ipython[notebook]==3.2 \
```
Comment (prabeesh) [inline Dockerfile:33]: latest version is fine here. Do you have any bug report to prove v4.0 is not supports profile correctly?

Comment (k-dahl) [inline Dockerfile:33]: Check on this one, unless I'm misunderstanding what he's saying:

https://github.com/jupyter/notebook/issues/309

Comment (prabeesh) [inline Dockerfile:33]: LGTM

File: docker-compose.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 web:
-    build: .
-    ports:
-        - "8888:8888"
-    working_dir:
-        - .:/notebook
+  build: ./spark
```
Comment (prabeesh) [inline docker-compose.yml:2]: Why do you move Dockerfile to /spark folder ?

Comment (k-dahl) [inline docker-compose.yml:2]: That's mainly for my own purposes as I intend to add some additional containers in the forked repo - I had only meant to do a pr for the first changes, but neglected to branch them first. I'll figure out a way to fix it.

**Comment by prabeesh** [comment]
@blitzd thanks for the fix.
If you wish to make a new PR please edit the Dockerfile without keeping it in a new folder.

**Pull request closed without merging**